dataset-spec.html

```diff
@@ -170,11 +170,11 @@ <h3><dfn>Dataset</dfn> interface</h3>
       Promise&lt;Dataset&gt;                  import (Stream stream);
       Dataset                           intersection (Dataset other);
       Dataset                           map (QuadMapIteratee iteratee);
-      Dataset                           merge ((Dataset or sequence&lt;Quad&gt;) quads);
       boolean                           some (QuadFilterIteratee iteratee);
       String                            toCanonical ();
       Stream                            toStream ();
       String                            toString ();
+      Dataset                           union ((Dataset or sequence&lt;Quad&gt;) quads);
```

Comment:
```suggestion
       Dataset                           union (Dataset quads);
```

Do you think we should make this consistent with `difference` and `intersection`? I imagine it'd be best to keep the interface for primitive set operations simple. Implementations can always overload for various parameters.